32blit-stm32/Src/SystemMenu/BatteryMenu.hpp

```diff
@@ -0,0 +1,33 @@
+/* BatteryMenu.hpp
+ * header file for Battery menu
+ * 
+ * The battery menu is a sub menu to show battery information
+ */
+
+#pragma once
+
+#include "engine/menu.hpp"
+
+class BatteryMenu final : public blit::Menu {
+public:
+    using blit::Menu::Menu;
+
+    //
+    // Prepare to show the battery menu
+    //
+    void prepare();
+
+    void render_footer();
```

Comment:
Hmm, Might be generally useful to have a virtual `render_footer` in the Menu class... Could call it from here-ish https://github.com/pimoroni/32blit-beta/blob/master/32blit/engine/menu.hpp#L48

Reply:
Great idea!

Reply:
Implemented this.